samplesheet_full.csv

```diff
@@ -0,0 +1,7 @@
+group,replicate,barcode,input_file,genome,transcriptome
+A549,1,,https://dl.dropboxusercontent.com/sh/q098af3xdzfqc72/AAD7YfS6uZNe5wz4Yuam46Dna/SGNex_A549_cDNA_replicate1_run2.fastq.gz,GRCh38,

```

Comment:
```suggestion
A549,1,,https://dl.dropboxusercontent.com/sh/q098af3xdzfqc72/AAD7YfS6uZNe5wz4Yuam46Dna/SGNex_A549_cDNA_replicate1_run2.fastq.gz,GRCh37,

```

Reply:
Maybe we should change this to `GRCh37` because `GRCh38` is currently an NCBI reference....

Reply:
Ok!